utils/goDotEnv.go

```diff
@@ -1,17 +1,17 @@
 package utils
 
 import (
+	"errors"
 	"github.com/joho/godotenv"
 	"log"
-	"errors"
 )
 
 func SetUpEnv() { //this is not for production environment only for dev, as we cannot load .ENV file in lambda
 
 	err := godotenv.Load("../.env")
 
 	if err != nil {
-		log.Println("Error loading env file")
+		log.Printf("Error loading env file %v", err)
```

Comment:
would prefer if errors could come in next line else it would be like
```
Error loading env file could not find the key
some unknown error
xyz
```

Reply:
Done